apps/src/lib/kits/maker/boards/microBit/MicroBitBoard.js

```diff
@@ -95,22 +94,17 @@ export default class MicroBitBoard extends EventEmitter {
   checkExpectedFirmware() {
     return Promise.resolve()
       .then(() => this.openSerialPort())
-      .then(serialPort => this.boardClient_.connectBoard(serialPort));
-  }
-
-  /**
-   * Initialize a set of component controllers.
-   * Exposed as a separate step here for the sake of the setup page; generally
-   * it'd be better to just call connect(), above.
-   * @throws {Error} if called before connecting to firmware
-   */
-  initializeComponents() {
-    return createMicroBitComponents(this.boardClient_).then(components => {
-      this.prewiredComponents_ = {
-        board: this.boardClient_,
-        ...components
-      };
-    });
+      .then(serialPort => this.boardClient_.connectBoard(serialPort))
+      .then(() => {
+        if (
+          this.boardClient_.firmwareVersion.includes('micro:bit Firmata 1.1')
```

Comment:
nit: 'micro:bit Firmata 1.1' should probably be a constant at the top of the file for easy visibility -- it feels a little hidden here